Tensile/ClientWriter.py

```diff
@@ -135,11 +131,57 @@ def main( config ):
 
   return returncode
 
+################################################################################
+# Write New Client Run Script
+################################################################################
+def writeNewClientRunScript(scriptPath, clientParametersPath, clientExePath = None): #, forBenchmark):
```

Comment:
I think we should avoid this pattern of creating scripts to run tasks.  We've been trying to avoid adding more code generation where possible.

If we want to be able to rerun the client, it should be exposed via a separate Python script or via a subcommand of the main Tensile script.